sherpa/ui/utils.py

```diff
@@ -320,6 +320,55 @@ def notice_data_range(get_data: Callable[[IdType], Data],
         report_filter_change(idstr, ofilter, nfilter, xlabel)
 
 
+@overload
+def calc_multiplot_size(rows: int,
+                        cols: Optional[int],
+                        nplots: int
+                        ) -> tuple[int, int]:
+    ...
+
+@overload
+def calc_multiplot_size(rows: Optional[int],
+                        cols: int,
+                        nplots: int
+                        ) -> tuple[int, int]:
+    ...
+
+def calc_multiplot_size(rows, cols, nplots):
+    """How many rows and columns to use for multi plot/contours?
+
+    Parameters
+    ----------
+    rows, cols
+       The requested sizes, if set (one of them must be set).
+    nplots
+       The nuber of plots.
```

Comment:
```suggestion
       The number of plots.
```

Reply:
These are from #2126 so once that has been merged (as I've just fixed them) there we can rebase or fix-up this PR to pick them up.